components/head.tsx

```diff
@@ -0,0 +1,13 @@
+import Head from 'next/head'
```

Comment:
there is a better way to do this (I might get to it before you), it's overriding the `[pages/_document.tsx](https://nextjs.org/docs/#custom-document)`